main.py

```diff
@@ -95,6 +95,19 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/queue/", response_model=Dict)
```

Comment:
Please write down the type annotation in more detail.

Reply:
E.g., `Dict[str, Any]` (please use the exact type, this is just an example).

Reply:
Is it resolved now?

Reply:
Umm.. how about `Dict[int, Dict[str, Any]]`?

Reply:
Okay, I will fix it that way.